aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -235,7 +241,15 @@ impl AptosVM {
     ) -> Self {
         let _timer = TIMER.timer_with(&["AptosVM::new"]);
         let randomness_api_v0_required_deposit = RequiredGasDeposit::fetch_config(resolver)
-            .unwrap_or_else(RequiredGasDeposit::default_if_missing);
+            .unwrap_or_else(RequiredGasDeposit::default_if_missing)
```

Comment:
This can also be factored out to that helper config file, VM only needs to know how to create a randomness config, but not the exact resources it has to fetch. Gas schedule is a good example for that.

Reply:
fixed